src/entities/dataset/dataset.ts

```diff
@@ -39,6 +39,12 @@ export class Dataset extends BaseEntity {
     @Column({ type: 'timestamptz', nullable: true })
     archive: Date;
 
+    @Column({ name: 'start_date', type: 'date', nullable: true })
+    startDate: Date | null;
+
+    @Column({ name: 'end_date', type: 'date', nullable: true })
+    endDate: Date | null;
```

Comment:
As above, do we need the `| null` or will `startDate?: Date` suffice?

Reply:
You need the null to be able unset the value.  I think this is an issue around TypeScript and of course undefined and null being different states in JavaScript.